harbors/schema.py

```diff
@@ -89,35 +86,28 @@ class Meta:
         ]
         geojson_field = "location"
         interfaces = (relay.Node,)
-        fields = (
-            "servicemap_id",
-            "zip_code",
-            "phone",
-            "email",
-            "www_url",
-            "image_link",
-            "zip_code",
-            "repair_area",
-            "electricity",
-            "water",
-            "gate",
-            "summer_storage_for_docking_equipment",
-            "summer_storage_for_trailers",
-            "summer_storage_for_boats",
-            "availability_level",
-            "number_of_marked_places",
-            "max_width",
-            "max_length",
-            "number_of_section_spaces",
-            "max_length_of_section_spaces",
-            "number_of_unmarked_spaces",
-            "location",
```

Comment:
looked at the linked release notes and PRs and cannot see why this change is needed 🤔  these fields are model fields, graphene should be able to convert them automatically. linked PRs were about fields, that are listed in graphene class' `Meta` but are not model fields 🤔

Reply:
Yup, the reason why I went on with this is because, otherwise, it raises the following error:

```
UserWarning: Field name "number_of_places" matches an attribute on Django model "harbors.Harbor" but it's not a model field so Graphene cannot determine what type it should be. Either define the type of the field on DjangoObjectType "HarborType" or remove it from the "fields" list.
```
with all the fields inherited from `AbstractArea`. The curious thing is that this only happens with resources on the `harbors` app, so I guess it could have something to do on how the schema is built? Since the "old" schema is built explicitly by `graphene.Schema`, but the "new" one is built through Federation... 🤔 

Without the explicit declaration, _it works_, but it throws a ton of warnings.

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/15201480/89177617-89ae5080-d594-11ea-918c-2da642a16480.png">

Reply:
ah, interesting, they are all passed as attributes from the abstract base model. well, then the change is ok 🙆‍♂️